template.yaml

```diff
@@ -740,20 +778,69 @@ Resources:
           - Fn::ImportValue: !Sub ${VpcStackName}-PrivateSubnetIdB
           - Fn::ImportValue: !Sub ${VpcStackName}-PrivateSubnetIdC
       Tags:
-        CheckovRulesToSkip: "CKV_AWS_115.CKV_AWS_117.CKV_AWS_173"
+        CheckovRulesToSkip: "CKV_AWS_115.CKV_AWS_116.CKV_AWS_117.CKV_AWS_173"
+
+  GroConvertToJsonFunctionLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      RetentionInDays: 14
```

Comment:
We probably need to have a global think about log retention at some point. 14 days feels pretty short to me